### DISCUSSION TITLE
FATAL:hwnd_util.cc(64)

### DISCUSSION THREAD

**Discussion by Robynetv69** [cefsharp/CefSharp#4708] [General]
Hello, has anyone come across this kind of error? It happens when with CEF I click on a link that has to open a kind of popup. 
If I look at the log file I see:

[0207/151243.678:FATAL:hwnd_util.cc(64)] 1400

I search over the internet but i don't find any suggestion.
Thanks in advance.
Roberto.

**Comment by amaitland** [comment]
https://source.chromium.org/chromium/chromium/src/+/main:ui/gfx/win/hwnd_util.cc;l=64?q=hwnd_util.cc&ss=chromium
https://magpcss.org/ceforum/viewtopic.php?f=6&t=17814
https://magpcss.org/ceforum/viewtopic.php?f=6&t=18551&start=10
https://github.com/cefsharp/CefSharp/issues/2378

Generally it suggests that an invalid parameter is being passed to the CreateWindowEx call.

See also https://source.chromium.org/chromium/chromium/src/+/main:ui/gfx/win/window_impl.cc;l=219;drc=7025c360a96319db787feb39a64f3a490e710bfe;bpv=1;bpt=1
